docs/history/changelog.md

```diff
@@ -101,6 +101,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
   * [#1403](https://github.com/open-contracting/standard/pull/1403) Replace "contract notice" with "contract signature notice", add synonyms to all notice codes ('plannedProcurementNotice', 'tenderNotice', 'awardNotice', 'contractNotice') and standardize the descriptions.
   * [#1415](https://github.com/open-contracting/standard/pull/1415) Change 'evaluationReports' section from "tender" to "award".
   * [#1416](https://github.com/open-contracting/standard/pull/1416) Change "interested supplier" to "potential supplier".
+  * [#1453](https://github.com/open-contracting/standard/pull/1453) Add 'market engagement notice'.
```

Comment:
Also, move this under the "unsuccessfulDetails" entry, so that all additions are together.

```suggestion
  * [#1453](https://github.com/open-contracting/standard/pull/1453) Add 'marketEngagementNotice'.
```